src/Tools/dotnet-dump/Analyzer.cs

```diff
@@ -147,10 +170,12 @@ public Task<int> Analyze(FileInfo dump_path, string[] command)
             }
             finally
             {
-                if (_target != null)
+                foreach (ITarget target in _targets.ToArray())
```

Comment:
Do we need the allocation of the array here?

Reply:
The _targets list gets updated when the target is destroy.  So I use ToArray to make a copy before the enumeration/destruction.